ospf6d/ospf6_abr.c

```diff
@@ -193,32 +193,34 @@ int ospf6_abr_originate_summary_to_area(struct ospf6_route *route,
 	if (route->type == OSPF6_DEST_TYPE_ROUTER) {
 		if (ADV_ROUTER_IN_PREFIX(&route->prefix)
 		    == area->ospf6->router_id) {
-			inet_ntop(AF_INET,
-				  &(ADV_ROUTER_IN_PREFIX(&route->prefix)), buf,
-				  sizeof(buf));
-			zlog_debug(
-				"%s: Skipping ASBR announcement for ABR (%s)",
-				__func__, buf);
+			if (is_debug) {
```

Comment:
The correct thing to do here is to determine (or create) the correct OSPF-level debug statement here and use that to determine whether or not this debugging code should be executed.